src/m365/spo/commands/list/list-roleinheritance-break.ts

```diff
@@ -0,0 +1,105 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { validation } from '../../../../utils';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+  clearExistingPermissions?: boolean;
+}
+
+class SpoListRoleInHeritanceBreakCommand extends SpoCommand {
+  public get name(): string {
+    return commands.LIST_ROLEINHERITANCE_BREAK;
+  }
+
+  public get description(): string {
+    return 'Break inheritance on list or library. Keeping existing permissions is the default behavior.';
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    if (this.verbose) {
+      logger.logToStderr(`breaking role inheritance of list in site at ${args.options.webUrl}...`);
+    }
+
+    let requestUrl: string = '';
+
+    if (args.options.listId) {
+      requestUrl = `${args.options.webUrl}/_api/web/lists(guid'${encodeURIComponent(args.options.listId)}')`;
+    }
+    else {
+      requestUrl = `${args.options.webUrl}/_api/web/lists/getbytitle('${encodeURIComponent(args.options.listTitle as string)}')`;
+    }
+
+    let clearExistingPermissions: boolean = true;
```

Comment:
Let's rename this variable to `keepExistingPermissions` so that it clearly depicts its value and purpose

Reply:
done